keps/sig-release/2853-k-core-branch-rename/kep.yaml

```diff
@@ -0,0 +1,58 @@
+title: Changing kubernetes/kubernetes default branch name to main
+kep-number: 2853
+authors:
+  - "@cpanato"
+owning-sig: sig-release
+participating-sigs:
```

Comment:
In the future I would bring a KEP to participating SIGs before merging it.
This will have impact well beyond SIG Release.